### PR TITLE
updated variables file to include uscranton

### PR DESCRIPTION
**Pull request by rlappel**
I thought I also created a dag. Maybe not.

### CODE REVIEW

File: variables.json [hunk 1]
```diff
@@ -117,12 +117,49 @@
     "schematron_filter": "validations/qdcingest_reqd_fields.sch",
     "schematron_report": "validations/padigital_missing_thumbnailURL.sch"
   },
-  "PENNSTATE_TARGET_ALIAS_ENV": "dev",
-  "PENNSTATE_XSL_CONFIG": {
+  "USCRANTON_TARGET_ALIAS_ENV": "dev",
```
Comment (htomren) [inline variables.json]: this seems to be writing over pennstate xsl config? I'm confused. Also, if Scranton is csv, there will be no OAI config. The variables would look more like what we have for Free Library.

File: variables.json [hunk 2]
```diff
@@ -117,12 +117,49 @@
...
+  "USCRANTON_XSL_CONFIG": {
     "schematron_filter": "validations/padigital_reqd_fields.sch",
     "schematron_report": "validations/padigital_missing_thumbnailURL.sch",
     "xsl_branch": "master",
     "xsl_filename": "transforms/pennstateingest.xsl",
```
Comment (htomren) [inline variables.json]: this is for penn state

File: variables.json [hunk 3]
```diff
@@ -117,12 +117,49 @@
...
+    "xsl_repository": "tulibraries/aggregator_mdx"
+      },
+  "USCRANTON_OAI_CONFIG": {
+    "endpoint": "http://digital.libraries.psu.edu/oai/oai.php",
```
Comment (htomren) [inline variables.json]: this is also penn state

File: variables.json (hunk 1 above)
Comment (lfinnigan) [reply variables.json]: Only the Internet Archive collection is CSV; the rest are from cdm

Comment (rlappel) [reply variables.json]: They only have one CSV collection. Let me fix this.

File: variables.json [hunk 4]
```diff
@@ -123,6 +123,43 @@
...
+      "p16214coll1",
+      "p9000coll4",
+      "p9000coll6",
+      "p9000coll7",
+      "passionists",
+      "prpubs",
+      "publications",
+      "scholarship",
+      "SCRYB03",
+      "zanerbloser",
```
Comment (lfinnigan) [inline variables.json]: the ending comma should be deleted after zanerbloser along with the extra new line below

**Pull request merged**